pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -416,3 +433,13 @@ func checkCRINetworkStats(assert *assert.Assertions, actual *statsapi.NetworkSta
 	assert.Equal(expected.Interfaces[0].TxBytes, *actual.TxBytes)
 	assert.Equal(expected.Interfaces[0].TxErrors, *actual.TxErrors)
 }
+
+func makeFakeLogs(sandbox *critest.FakePodSandbox, containers ...string) {
+	for _, name := range containers {
+		os.MkdirAll(fmt.Sprintf("/var/log/pods/%s/%s/", types.UID(sandbox.GetMetadata().GetUid()), name), 755)
```

Comment:
`/var/log/pods` actual pod/container log directly. We should not create files there in unit test...

Reply:
yep went mock approach